File: C#/OOP/Prvni/Accountant.cs
using System;

//v tomhle jsem měl trochu bordel, kdyžtak checknout doma

namespace Prvni12 {
    class Accountant : Employee {

        public Accountant() {

        }
        public Accountant(int age, int salary) : base(age, salary) {
            //this.age = age;   toto a druhy řádek nahrazuje dědění kontruktoru base(age,salary)
            //this.salary = salary;
        }

        public override void writeInfo() {
            base.writeInfo();

            Console.WriteLine();
        }
    }
}

File: Java/OOP/project1/src/project/Accountant.java
package project;

class Accountant extends Employee {

    public Accountant() {

    }
    public Accountant(int age, int salary) { super (age, salary); 
        //this.age = age;   toto a druhy øádek nahrazuje dìdìní kontruktoru base(age,salary)
        //this.salary = salary;
    }

    public void writeInfo() {
        super.writeInfo();

        System.out.println();
    }
}
